### ISSUE TITLE
Move Entities Issue

### ISSUE THREAD

**Issue by alefumero** [msummers123/Movecraft-3#161]
I´m not completly sure because i didnt find up to date documentation, but move entitie config isnt working att all, my airship doesnt move any entity nor droped items or mobs. 
Is this to be expected or a bug with the plugin or a setup error from my side?
Thanks for your help

**Comment by msummers123** [comment]
Yeah, currently move entity only refers to players. I took out the other items to resolve a bug, although I'm thinking I might add at least mobs back in.
